packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -155,7 +155,7 @@ export class Tooltip extends React.Component<ITooltipProps, {}> {
                 interactionKind={PopoverInteractionKind.HOVER_TARGET_ONLY}
                 lazy={true}
                 popoverClassName={classes}
-                transitionDuration={200}
+                transitionDuration={transitionDuration}
```

Comment:
the better thing to do is to remove this line entirely cuz it's covered by `{...this.props}` spread